packages/components/text/src/Variants.win32.ts

```diff
@@ -3,6 +3,8 @@ import { Text } from './Text';
 export const Caption1 = Text.customize({
   variant: 'caption1',
 });
+export const Caption1Strong = Text; // Not supported on Win32
```

Comment:
If they're not supported, would it make more sense to return null?

Reply:
Good idea, and it's consistent with [this comment](https://github.com/microsoft/fluentui-react-native/pull/2141#issuecomment-1252889912).

Resolved with fa48f6d.